utils/wasm-builder/src/wasm_project.rs

```diff
@@ -258,90 +231,13 @@ impl WasmProject {
             )
         })?;
 
-        // Write metadata
-        if let Some(metadata) = &self.project_type.metadata() {
-            let file_base_name = self
-                .file_base_name
-                .as_ref()
-                .expect("Run `WasmProject::create_project()` first");
-
-            let wasm_meta_path = self
-                .wasm_target_dir
-                .join([file_base_name, ".meta.txt"].concat());
-
-            smart_fs::write_metadata(wasm_meta_path, metadata)
-                .context("unable to write `*.meta.txt`")?;
-
-            source_code = format!(
-                r#"{source_code}
-#[allow(improper_ctypes)]
-mod fake_gsys {{
-    extern "C" {{
-        pub fn gr_reply(
-            payload: *const u8,
-            len: u32,
-            value: *const u128,
-            err_mid: *mut [u8; 36],
-        );
-    }}
-}}
-
-#[unsafe(no_mangle)]
-extern "C" fn metahash() {{
```

Comment:
are all metahash entries removed as well?

Reply:
yes, everything is removed except the api and runtime methods